### DISCUSSION TITLE
PTE & Custom Range

### DISCUSSION THREAD

**Discussion by JeremyVyska** [Spare-Brained-Community/Spare-Brained-Licensing#2] [General]
Hey all!

The plan is to pop the source code into this repo, renumbered into the 50-99k custom object range, to ensure your dev license can make use of it.  (For On-Prem, I can also do the Runtime .app in the licensed ISV range).

So - the challenge that I want some feedback on, what Custom Range would folks like it renumbered to?   For on-prem, if it's in 50000..50009, then it's likely no license changes are needed, but the object collision likelihood is super high.

Thoughts?

**Comment by waldo1001** [comment]
A license app in the 50k range … does that make sense?  I would only see licensing as part of a product, which is not in the 50k range, right?  Hence: should always be in an ISV range .. hence renumbering should always be done..? 🤔

(just thinking out loud here…)

**Comment by JeremyVyska** [comment]
You're not wrong - sitting in the custom object range lessens the ability it gives to have security be effective.

But, if people want to build and test with it, it either has to be in the custom range or I have to distribute my license with insert rights. 😄

For actual deployments, yes, it should not remain in the custom range.  Any changes made to this repo that gets incorporated, I'll bring those back into my 71mil copy, and republish the new AppSource and runtime versions.

Which sort of answers my question - no one should use the objects in this repo for production, so why worry about what collisions happen in Development?  Fair play.  🥰

**Comment by waldo1001** [comment]
True :-).

Well - people testing it might have to happen with their license, in combo with their app, which either would be in their ISV range, or in 50000 - so I'd just make sure it's outside of the 50000 range, like the 67k range?

**Comment by JeremyVyska** [comment]
Renumbering done.  Fingers crossed.
